guides/manual-installation.md

```diff
@@ -469,6 +469,69 @@ Expected output if Trento web/wanda is ready and the database connection is setu
 {"ready":true}{"database":"pass"}
 ```
 
+## Integrations
+
+### OpenID Connect
+
+Trento integrates with an identity provider (IDP) that uses the OpenID Connect (OIDC) protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
+
+
+#### User Roles and Authentication
+
+User authentication is entirely managed by the IDP, which is responsible for maintaining user accounts. 
+A user, who does not exist on the IDP, is unable to access the Trento web console.
+If the default admin user created during the installation process matches the authenticated user’s IDP username, that user is automatically granted `all:all` permissions within Trento.
```

Comment:
I would clarify here that this exactly happens when the `ADMIN_USERNAME` variable matches.
Maybe like:

```
If the default admin user created during the installation process (using the `ADMIN_USER` variable, `admin` being the default value) matches the authenticated user....
```